packages/rs-drive-abci/src/execution/platform_events/block_end/validator_set_update/v1/mod.rs

```diff
@@ -0,0 +1,216 @@
+use crate::error::execution::ExecutionError;
+use crate::error::Error;
+use crate::execution::types::block_execution_context::v0::{
+    BlockExecutionContextV0Getters, BlockExecutionContextV0MutableGetters,
+};
+use crate::execution::types::block_execution_context::BlockExecutionContext;
+use crate::platform_types::platform::Platform;
+use crate::platform_types::platform_state::v0::PlatformStateV0Methods;
+use crate::platform_types::platform_state::PlatformState;
+use crate::platform_types::validator_set::v0::ValidatorSetV0Getters;
+use crate::rpc::core::CoreRPCLike;
+use itertools::Itertools;
+
+use dpp::dashcore::hashes::Hash;
+use tenderdash_abci::proto::abci::ValidatorSetUpdate;
+
+impl<C> Platform<C>
+where
+    C: CoreRPCLike,
+{
+    /// We need to validate against the platform state for rotation and not the block execution
```

Comment:
This confused me. It might be good to annotate the comments like "V0 comment: ..." and then "V1 comment: ..."